### ISSUE TITLE
Investigate CI Failure - entrypoint interpolation

### ISSUE THREAD

**Issue by karlkfi** [mesosphere/kubernetes-mesos#608]
```
[10:43:44][Step 4/4] Invalid interpolation format for "entrypoint" option in service "mesosslave": "NAME=$(cut -f2 -d/ <<<${MESOSMASTER1_NAME}) && N=${NAME##*_} && DOCKER_NETWORK_OFFSET=0.0.${N}.0 exec wrapdocker mesos-slave --work_dir="/var/tmp/mesos/${N}" --attributes="rack:${N};gen:201${N};role:role${N}" --hostname=$(getent hosts mesosslave | cut -d' ' -f1 | sort -u | tail -1) --resources="cpus:4;mem:1280;disk:25600;ports:[8000-21099];cpus(role${N}):1;mem(role${N}):640;disk(role${N}):25600;ports(role${N}):[7000-7999]"
```

https://teamcity.mesosphere.io/viewLog.html?buildId=67506&buildTypeId=Oss_KubernetesMesos_5SmokeTestsDockerMesos&tab=buildLog&guest=1

**Comment by karlkfi** [comment]
Looks like it's a regression or change of behavior in docker-compose 1.5.0 which is being installed by `pip install -U docker-compose` in the test image on hosts that don't have that image cached.

I'm going to hardcode to 1.4.2 until I figure out why it's breaking.

**Comment by karlkfi** [comment]
Correction... same problem in 1.4.2

**Comment by karlkfi** [comment]
https://github.com/docker/compose/releases

```
Breaking changes:

With the introduction of variable substitution support in the Compose file, any Compose file that uses an environment variable ($VAR or ${VAR}) in the command: or entrypoint: field will break.

Previously these values were interpolated inside the container, with a value from the container environment. In Compose 1.5.0, the values will be interpolated on the host, with a value from the host environment.

To migrate a Compose file to 1.5.0, escape the variables with an extra $ (ex: $$VAR or $${VAR}). See
https://github.com/docker/compose/blob/8cc8e61/docs/compose-file.md#variable-substitution
```

**Comment by karlkfi** [comment]
Ok, not going to revert docker-compose. Need to fix the interpolation for 1.5.0 and then i'll fix the compose version to avoid auto-upgrading in the future.
